site-src/api-types/backendtlspolicy.md

```diff
@@ -28,6 +28,16 @@ to prevent the complications involved with sharing trust across namespace bounda
 
 All Gateway API Routes that point to a referenced Service should respect a configured BackendTLSPolicy.
 
+## Gateway Backend TLS Configuration
```

Comment:
@08volt This actually belongs under the Gateway "API Types" reference more than the BackendTLSPolicy "API Types" reference.  If you decide to keep it in both places, make sure to point out whether this Gateway field can be overriden any BackendTLSPolicy in effect.  See https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1/gateway_types.go#L507